src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -303,7 +304,7 @@ private function autocomplete(OutputInterface $output, Question $question, $inpu
                 // Save cursor position
                 $output->write("\0337");
                 // Write highlighted text
-                $output->write('<hl>'.substr($matches[$ofs], $i).'</hl>');
+                $output->write('<hl>'.OutputFormatter::escapeTrailingBackslash(substr($matches[$ofs], $i)).'</hl>');
```

Comment:
cant we use `escape()` here (it includes `escapeTrailingBackslash` also)? Not sure about leaking out this detail =/

Reply:
As noted in the PR description, I think this is a BC break as `escape` would effectively turn off interpreting tags in autocompletion options (they'll be always shown as is); while the impact should not be terrible, this doesn't seem to be the right thing to do for anything but master where it could be acceptable.

Reply:
Right! 

> they'll be always shown as is

Which is in fact the actual value.. no? But :+1: for keeping as is; thus escapeTrailingBackslash only.

Reply:
@ro0NL , yeah.
I'd be glad to create another one for master with just `escape` as it indeed seems to be the right thing in general (interpretation of tags is not guaranteed there anyway, only `<hl>` accidentally works).

Reply:
unless we qualify it a bugfix for 2.7 ;-) impact should be minimal. More or less relates to escaping exceptions https://github.com/symfony/symfony/pull/22142

also `h1` leaks in as a valid style after calling... perhaps inline it using background/foreground options.

Reply:
I would keep it as is unless someone asks for it, escaping trailing backslashes only